tools/windows/DatadogAgentInstaller/WixSetup/Datadog Agent/AgentInstaller.cs

```diff
@@ -591,7 +594,28 @@ private Dir CreateBinFolder()
                     AttributesDefinition = "SupportsErrors=yes; SupportsInformationals=yes; SupportsWarnings=yes; KeyPath=yes"
                 },
                 agentBinDir,
-                new WixSharp.File(_agentBinaries.LibDatadogAgentThree)
+                new WixSharp.File(_agentBinaries.LibDatadogAgentThree),
+                new WixSharp.File(@"C:\opt\datadog-installer\datadog-installer.exe",
+                    new ServiceInstaller
+                    {
+                        Name = "Datadog Installer",
+                        StartOn = SvcEvent.Install,
+                        StopOn = SvcEvent.InstallUninstall_Wait,
```

Comment:
should use our custom actions for this, see comment on other service defs for details